src/components/MessagesList/MessagesGroup/Message/MessagePart/MessageBody.vue

```diff
@@ -26,6 +26,9 @@
 
 			<!-- Additional controls -->
 			<CallButton v-if="showJoinCallButton" />
+			<CalendarEventConversationActions v-if="showEventConversationActions"
```

Comment:
better to be `v-else-if` here, all three do not have overlaping conditions, and only one at the time can be rendered